frontend/public/components/deployment-config.tsx

```diff
@@ -262,6 +264,29 @@ export const DeploymentConfigDetailsList = ({ dc }) => {
   );
 };
 
+export const DeploymentConfigDeprecationAlert: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      isInline
+      variant="info"
+      title={t('public~DeploymentConfigs is being deprecated with OpenShift 4.14')}
+    >
+      <p>{t('public~Feature development of DeploymentConfigs will be deprecated in OpenShift Container Platform 4.14.')}</p>
+      <p>
+        {t(
+          'public~DeploymentConfigs will continue to be supported for security and critical fixes, but you should migrate to Deployments wherever it is possible.',
+        )}
+      </p>
+      <ExternalLink
+        href="https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/building_applications/deployments#doc-wrapper"
```

Comment:
We should be using the `getDocumentationURL()` function.
Check:
- https://github.com/openshift/console/blob/master/frontend/public/components/build.tsx#L237
- https://github.com/openshift/console/blob/master/frontend/public/components/utils/documentation.tsx#L87

This URL would refer to downstream docs link. For upstream I would use `applications/deployments/what-deployments-are.html` path